sap/cli/checkin.py

```diff
@@ -473,7 +473,7 @@ def do_checkin(connection, args):
             console.printout('Creating objects ...')
             inactive_objects = _checkin_dependency_group(connection, activation_group, console, args.corrnr)
 
-            if inactive_objects.references is not None:
+            if inactive_objects.references:
```

Comment:
Could you please explain in the commit message when the condition is met and why the check for None was OK before?